### ISSUE TITLE
[Feature Request] withoutTags scope, reverse of withAllTags

### ISSUE THREAD

**Issue by GRMule** [rtconner/laravel-tagging#115]
My use case: a CMS that displays lists of content from a central content pool, filtered by tag and/or content type.

You might use a tag like "Ad" on some banner images, which you would show in a rotating list filtered by that tag, say something for sidebars and the like. You wouldn't want those ad posts showing up in, for example, a list of newly posted images. So you'd exclude that tag from the query for the page showing recently posted media.

``` php
    $filterTags = array('Nature');
    $query->withAllTags($filterTags);

    $excludeTags = array('Ads', 'NSFW');
    $query->withoutTags($excludeTags);

    $query->limit(8);

    // result: 8 Contents tagged with "Nature" NOT tagged with "Ads" or "NSFW"
```

**Comment by msieprawski** [comment]
Any updates on this one?

**Comment by rtconner** [comment]
Resolved in latest release #124

**Comment by GRMule** [comment]
Nice! Thanks!
